src/main/java/cd/go/contrib/elasticagent/PluginRequest.java

```diff
@@ -86,4 +90,23 @@ public void deleteAgents(Collection<Agent> toBeDeleted) throws ServerRequestFail
             throw ServerRequestFailedException.deleteAgents(response);
         }
     }
+
+    public void appendToConsoleLog(JobIdentifier jobIdentifier, String text) throws ServerRequestFailedException {
+        Map<String, String> requestMap = new HashMap<>();
+        requestMap.put("pipelineName", jobIdentifier.getPipelineName());
```

Comment:
Gah. Did we use camelCase, @arvindsv?

Reply:
😳 I guess so. :( Sorry. I'll introduce a new version.